src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/CoordinatesFormattingTest.java

```diff
@@ -0,0 +1,69 @@
+/* (c) 2018 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.test;
+
+import org.geoserver.catalog.FeatureTypeInfo;
+import org.geotools.feature.NameImpl;
+import org.junit.Test;
+import org.w3c.dom.Document;
+
+/** Testing for Coordinates formatting configurations on WFS GML 3.1 & 3.2 on complex features */
```

Comment:
This issue associated to this PR contains in its description:

- allow / deny scientific notation
- force right-zero padding up to configured # of decimals

Unless I missed them, I don't think there is a test ``allow / deny scientific notation``.

Reply:
Good point, adding